artiq/test/test_rpctool.py

```diff
@@ -0,0 +1,38 @@
+from artiq.protocols.pc_rpc import Server
+import asyncio
+import sys
+import unittest
+
+class Target:
+    def message(self):
+        return 4
+
+class Rpctool_test(unittest.TestCase):
+    async def get_message(self):
```

Comment:
This is not just getting the message, it is also checking its value. Rename that function, or just merge it into the ``do_test`` function below.